notebook/static/notebook/js/outputarea.js

```diff
@@ -783,6 +787,18 @@ define([
         element.append(toinsert);
         return toinsert;
     };
+    
+    var append_mathml = function (mathml, md, element) {
+        /**
+         * This method cannot do the typesetting because the mathml first has to
+         * be on the page.
+         */
+        var type = 'application/mathml+xml';
+        var toinsert = this.create_output_subarea(md, "output_mathml", type);
+        toinsert.text(mathml);
```

Comment:
Ok it looks like `append_html` or equivalents use `toinsert.append()`, so it would look like:

```js
var append_mathml = function (mathml, md, element) {
        var type = 'application/mathml+xml';
        var toinsert = this.create_output_subarea(md, "output_mathml", type);
        toinsert.append(mathml);
        element.append(toinsert);
        return toinsert;
};
```

How do we go about sanitizing the HTML?